woocommerce/woocommerce-ios
# [Mobile Payments] Add (empty) knownReaders

Closes #4047 

Note: This is against feature/stripe-terminal-sdk-integration, not develop

- Adds a `loadKnownReaders` action to our CardPresentPaymentsStore. We'll need this for the viewmodel to be added in #4049
- Note:  Returns an empty array presently. Actually "knowing" and "unknowing" readers will come in #3559 

To test:
- No user facing functionality
- Simply confirm all unit tests continue to pass

Update release notes:
